apps/shipments/views.py

```diff
@@ -154,7 +154,7 @@ def update(self, request, *args, **kwargs):
         log_metric('transmission.info', tags={'method': 'shipments.update', 'module': __name__})
 
         serializer = ShipmentUpdateSerializer(instance, data=request.data, partial=partial,
-                                              context={'auth': get_jwt_from_request(request)})
+                                              context={'auth': get_jwt_from_request(request), 'shipment': instance})
```

Comment:
You don't need to pass the an object's instance to that object's serializer via context, it's already part of the serializer.